### PR TITLE
BATIAI-2324: Create roles for gitlab-runner cosign  in ub-irsa

### PR DESCRIPTION
**Pull request by skumbala40**
### Submitter Checklist
-  [ ] Is there an impact on Auditing and Logging procedures or capabilities?
-  [ ] Is there an impact on Authentication procedures or capabilities?
-  [ ] Is there an impact on Authorization procedures or capabilities?
-  [ ] Is there an impact on Communication Security procedures or capabilities?
-  [ ] Is there an impact on Cryptography procedures or capabilities?
-  [ ] Is there an impact on Sensitive Data procedures or capabilities?
-  [ ] Is there an impact on any other security-related procedures or capabilities?
-  [*] No security impacts identified.

Security Risks Identified - For any applicable items on the "Submitter Checklist," describe the impact of the change and any implemented mitigations.

### CODE REVIEW

**Comment by robo-gotham** [comment]
Snyk Scanning for Commit: cc493d6c6a060bee0eedc73d500b3db59879e865

Snyk Infrastructure as Code

- Snyk testing Infrastructure as Code configuration issues.
✔ Test completed.

Issues
  No vulnerable paths were found!

-------------------------------------------------------

</details>

Test Summary

  Organization: batcave-ispg
  Project name: CMS-Enterprise/batcave-tf-utility-belt-irsa

✔ Files without issues: 3
✗ Files with issues: 0
  Ignored issues: 0
  Total issues: 0 [ 0 critical, 0 high, 0 medium, 0 low ]

-------------------------------------------------------

Report Complete

  Your test results are available at: https://snyk.io/org/batcave-ispg/projects
  under the name: CMS-Enterprise/batcave-tf-utility-belt-irsa

File: variables.tf [hunk 1]
```diff
@@ -154,3 +154,19 @@ variable "grafana_service_accounts" {
...
+  type        = bool
+}
+
+variable "iam_role_path" {
+  default = "/delegatedadmin/developer/"
+  type    = string
+}
+
+variable "iam_role_permissions_boundary" {
+  default = "arn:aws:iam::373346310182:policy/cms-cloud-admin/developer-boundary-policy"
```
Comment (bushong1) [inline variables.tf:170]:
```suggestion
```

Comment (bushong1) [reply variables.tf:170]: I know this is set as default elsewhere, but it probably shouldn't be.  We won't be able to create roles unless this is set.  If we decide to keep a default, we should set it as empty string.

File: variables.tf [hunk 2]
```diff
@@ -154,3 +154,19 @@ variable "grafana_service_accounts" {
...
 }
+
+variable "create_ub_cosign_iam_role" {
+  description = "Flag to create Cosign IAM role"
+  default     = false
+  type        = bool
+}
+
+variable "iam_role_path" {
+  default = "/delegatedadmin/developer/"
```
Comment (bushong1) [inline variables.tf:165]:
```suggestion
  default = ""
```

File: variables.tf (hunk 1 above)
Comment (skumbala40) [reply variables.tf]: Updated as suggested.

File: variables.tf (hunk 2 above)
Comment (skumbala40) [reply variables.tf]: Updated as suggested.

**Review by bushong1 — APPROVED** [review]

**Pull request merged**